counsel.el

```diff
@@ -4191,7 +4191,7 @@ When ARG is non-nil, display all active evil registers."
       (let* ((counsel--mark-ring-calling-point (point))
              (candidates (counsel-mark--get-evil-candidates arg)))
         (if candidates
-            (counsel-mark--ivy-read candidates 'counsel-evil-marks)
+            (counsel-mark--ivy-read "Evil marks: " candidates 'counsel-evil-marks)
```

Comment:
Just curious: how is an "evil mark" different to a normal marker?

Reply:
Evil marks are like vim marks, their tied to a specific character so in emacs nomeclature an evil-mark is a register-mark. For example I can hit `ma` at any point and now there's a mark stored in that character, I can run `'a` to jump to the mark I placed. `counsel-evil-marks` just shows all those dropped marks to you in an ivy session.

Evil also has some special marks that're automatically changed as you edit (eg. `'<` points to the start of a region/visual-selection).
![Screenshot_20201223_183845](https://user-images.githubusercontent.com/23294780/103027715-4ad26480-454e-11eb-9141-2c763a8c607a.png)